Packs/Phishing/Playbooks/Process_Email_-_Generic_v2.yml

```diff
@@ -2957,6 +2956,46 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "74":
+    id: "74"
+    taskid: e8aecf09-e057-48c2-8fcb-d67de5afbc91
+    type: regular
+    task:
+      id: e8aecf09-e057-48c2-8fcb-d67de5afbc91
+      version: -1
+      name: Read QR Code from image files
+      description: Extracts the text from a QR code. The output of this script includes the output of the script "extractIndicators" run on the text extracted from the QR code.
```

Comment:
Recommend:
Extracts the text from a QR code. The output of this task includes the output of the extractIndicators script that runs on the text extracted from the QR code.